### ISSUE TITLE
SPLIT_CONTRACTIONS missing cases issue

### ISSUE THREAD

**Issue by radamchin** [dhowe/RiTaJS#9]
I've run into an issue of "didn't" being auto split to "did" "not" when using RiTa.tokenize(),
even when RiTa.SPLIT_CONTRACTIONS == false

It appears this case & a few others did not make it into the SPLIT_CONTRACTIONS if clause of tokenize() method.

See:  https://github.com/dhowe/RiTaJS/blob/master/src/rita.js#L290-L303 

```
   if (RiTa.SPLIT_CONTRACTIONS) { // SAVE
      words = words.replace(/'ll /g, " 'll ");
      words = words.replace(/'re /g, " 're ");
      words = words.replace(/'ve /g, " have ");
      words = words.replace(/n't /g, " not ");
      words = words.replace(/'LL /g, " 'LL ");
      words = words.replace(/'RE /g, " 'RE ");
      words = words.replace(/'VE /g, " 'VE ");
      words = words.replace(/N'T /g, " N'T ");
    }
// These conditions below \/ should be inside this /\ ??
    words = words.replace(/ ([Cc])an't /g, " $1an not ");
    words = words.replace(/ ([Cc])annot /g, " $1an not ");
    words = words.replace(/ ([Dd])idn't /g, " $1id not ");
    words = words.replace(/ ([CcWw])ouldn't /g, " $1ould not ");
```

**Comment by dhowe** [comment]
Looking into it.

**Comment by dhowe** [comment]
Should be fixed into v1.1.2 -- please verify

**Comment by radamchin** [comment]
Fixed. Cheers
